src/models/neural-sentiment-analysis.ts

```diff
@@ -0,0 +1,36 @@
+import Attributes from './attributes';
+import RestfulModel from './restful-model';
+
+export default class NeuralSentimentAnalysis extends RestfulModel {
+  account_id?: string;
+  sentiment?: string;
+  sentimentScore?: number;
+  processedLength?: number;
+  text?: string;
+
+  fromJSON(json: object): this {
```

Comment:
how about 
```suggestion
  fromJSON(json: unknown): this {
```

Reply:
or typeof this

Reply:
@sdwvit just curious, why going from typing object to unknown?

Reply:
object is as broad as any in this case. Unknown just means you don't know the type. 
Not the best idea to have either of those. 

How about: 
```suggestion
  fromJSON(json: Partial<typeof this>): this {
```

Reply:
or anything like that, that imrpoves type

Reply:
I love the suggestion! Actually it turns out I don't even need this piece of code, it's redundant all it does is call the super method and the class `extends RestfulModel` so there's no need for it.